doc/install/build_install.rst

```diff
@@ -736,6 +761,42 @@ Putting it all together, a typical workflow to build Arbor on a Cray system is:
     Often the library or executable will work, however if a different glibc is loaded,
     Arbor will crash at runtime with obscure errors that are very difficult to debug.
 
+For building Arbor's python interface, it may be necessary to load a more recent
```

Comment:
May be, or is?

Reply:
Haven't tested all systems, so cannot tell you exactly. It is required on Piz Daint, though. This isn't supposed to be an exhaustive system documentation, and people are expected to look up their system specs / best practices / ask sysadmin etc.

Reply:
I guess 'knows about the Cray system' is the operating clause, not 'more recent Python'? Maybe:

```suggestion
For building Arbor's Python interface, it may be necessary to load a Python that knows about the target system; in the case of the CSCS Cray system:
```